feats/feats.yml

```diff
@@ -383,7 +383,7 @@
     You are specialized in the use a particular boon that is iconic to your character.
   effect: |
     <ul>
-    <li><strong>Tier 1</strong> - When you invoke the chosen boon on a single target, you succeed automatically and do not need to make an action roll. If the invocation has multiple targets, success is not automatic, but you get advantage 2 on the action roll to invoke the boon.</li>
+    <li><strong>Tier 1</strong> - You can invoke the boon at any of the power levels you could access via your other means. When you invoke the chosen boon on a single target, you succeed automatically and do not need to make an action roll. If the invocation has multiple targets, success is not automatic, but you get advantage 2 on the action roll to invoke the boon.</li>
```

Comment:
I know that's what you meant, but when I reread it I realized it broke up the sentence flow. It flows from describing a single-target to describing multi-target nicely, and the phrase being placed prior to both has the same effect without breaking up the flow.
If you still disagree I'm happy to change it, but that was my rational.